community/planning/dlt_polling_monitor.md

```diff
@@ -0,0 +1,110 @@
+<!--
+  Copyright 2022 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+# DLT Polling Monitor
+
+## Overview
+
+The **DLT Polling Monitor** is a utility that updates pending batches in Grid
+with the latest information from the DLT. It does this by polling the database
+and the `/batch_statuses` endpoint.
+
+## Input / Output Definition
+
+The DLT polling monitor uses the following inputs:
+
+- `poll_interval` (duration) - This is read from the environment, but has a
+  default value built-in value. We will want to change the poll interval based
```

Comment:
probably just intended to have either "default value" or "built-in value"

Reply:
Ah, good catch! Fixed